server/src/visitors/YmlBaseVisitor.ts

```diff
@@ -267,6 +257,9 @@ export class YmlBaseVisitor extends AbstractParseTreeVisitor<void> implements Ym
      * Do nothing if there is already a one space distance or if the two elements are not on the same line.
      */
     private setOneSpaceInterval(leftElementEnd: number, rightElementStart: number, sameLine: boolean): void {
+        if (this.isDocumentFormatImpossible()) {
```

Comment:
Not a huge fan of the fact that now we spam calls to functions that may end up doing nothing (thus making their names lie, in some way), but I suppose you didn’t move these checks around just for sport.

Reply:
I don't completely remember why and when I moved these instructions.
I guess it was to avoid copypasting calls to `this.isDocumentFormatImpossible()` in every `visitXXX` functions :thinking: